ModAssistant/Localisation/sv.xaml

```diff
@@ -0,0 +1,232 @@
+    </Span>
+    <Span x:Key="Intro:WikiGuide">
+        Var vänlig läs nybörjarguiden på
+        <Hyperlink local:HyperlinkExtensions.IsExternal="True" NavigateUri="https://bsmg.wiki/pc-modding.html">
+            Wikisidan
+        </Hyperlink>.
+    </Span>
+    <sys:String x:Key="Intro:AgreeButton">Godkänner</sys:String>
+    <sys:String x:Key="Intro:DisagreeButton">Nekar</sys:String>
+    <sys:String x:Key="Intro:ClosingApp">Stänger applikationen: Du godkände ej villkoren.</sys:String>
+    <sys:String x:Key="Intro:VersionDownloadFailed">Det gick inte att hämta listan över versioner.</sys:String>
+    <sys:String x:Key="Intro:ModsTabDisabled">Modfliken otillgänglig. Var vänlig starta om för att försöka igen.</sys:String>
+    <sys:String x:Key="Intro:ModsTabEnabled">Du har nu tillgång till Modfliken!</sys:String>
+
+    <!-- Mods Page -->
+    <sys:String x:Key="Mods:Title">Mods</sys:String>
+    <sys:String x:Key="Mods:Header:Name">Namn</sys:String>
+    <sys:String x:Key="Mods:Header:Installed">Installerad</sys:String>
+    <sys:String x:Key="Mods:Header:Latest">Senaste</sys:String>
+    <sys:String x:Key="Mods:Header:Description">Beskrivning</sys:String>
+    <sys:String x:Key="Mods:Header:Uninstall">Avinstallera</sys:String>
+    <sys:String x:Key="Mods:UninstallButton">Avinstallera</sys:String>
+    <sys:String x:Key="Mods:LoadFailed">Det gick inte att ladda modlistan</sys:String>
+    <sys:String x:Key="Mods:CheckingInstalledMods">Kollar efter installerade mods</sys:String>
+    <sys:String x:Key="Mods:LoadingMods">Laddar Mods</sys:String>
+    <sys:String x:Key="Mods:FinishedLoadingMods">Mods färdigladdade!</sys:String>
+    <sys:String x:Key="Mods:InstallingMod">Installerar {0}</sys:String>
+    <sys:String x:Key="Mods:InstalledMod">Installerat {0}</sys:String>
+    <sys:String x:Key="Mods:FinishedInstallingMods">Mods färdiginstallerade!</sys:String>
+    <sys:String x:Key="Mods:ModDownloadLinkMissing">Det gick inte att hitta någon nedladdningslänk för {0}</sys:String>
+    <sys:String x:Key="Mods:UninstallBox:Title">Avinstallera {0}?</sys:String>
+    <sys:String x:Key="Mods:UninstallBox:Body1">Är du säker på att du vill ta bort {0}?</sys:String>
+    <sys:String x:Key="Mods:UninstallBox:Body2">Detta kan leda till att dina andra mods slutar att fungera</sys:String>
+    <sys:String x:Key="Mods:FailedExtract">Extrahering misslyckades {0}, försöker igen om {1} sekunder. ({2}/{3})</sys:String>
+    <sys:String x:Key="Mods:FailedExtractMaxReached">Misslyckades med att extrahera {0} efter maxantalet försök ({1}), hoppar över. Denna mod kanske inte fungerar som den ska så fortsätt på egen risk.</sys:String>
+
+    <!-- About Page -->
+    <sys:String x:Key="About:Title">Om</sys:String>
+    <sys:String x:Key="About:PageTitle">Om Mod Assistant</sys:String>
+    <sys:String x:Key="About:List:Header">I'm Assistant, and I made Mod Assistant for mod assistance, med några grundläggande principer i åtanke:</sys:String>
```

Comment:
`I'm Assistant, and I made Mod Assistant for mod assistance, ...`
This appears to be untranslated. Is this right?

Reply:
This one is a bit difficult. It'll lose all it's playfulness if translated since it's basically a line that only makes sense in English.

It'd be "Jag är assistenten, och jag skapade Mod Assistant för modassistans, med några grundläggande principer i åtanke." - it works, but it sounds a little like a google translated kids show. c:

Reply:
Ha, fair enough. It's no problem, it just stood out to me!